### ISSUE TITLE
Issue on page /intro.html

### ISSUE THREAD

**Issue by Filipi-Soares** [zedomel/worldfair-agrobio#4]
@zedomel I was thinking if adding an example including before/after of one dataset that has been through the FAIRification Process would add some value to the cookbook. Maybe taking [calvalheiro2023](https://github.com/globalbioticinteractions/calvalheiro2023/archive/20cc192510e0abb7c982a50502354fa74d504cfa.zip) dataset as example. My idea is to show how it was when she shared it with us (original dataset), then the output of the processing by REBIPP, GloBI, and GBIF. Please let me know if this makes sense for you.

**Comment by zedomel** [comment]
Yes. I complete agree with you. Can you prepare such before/after example?
It can be created on a Google Docs and then I convert to markdown.
